libglusterfs/src/mem-pool.c

```diff
@@ -150,9 +150,8 @@ gf_mem_update_acct_info(struct mem_acct *mem_acct, struct mem_header *header,
 }
 
 static bool
-gf_mem_acct_enabled(void)
+gf_mem_acct_enabled(xlator_t *x)
```

Comment:
I think it would be better to use `xl` or `this`, which are the common names we use for xlator variables.

Reply:
Done, thanks.